test_rmw_implementation/test/test_subscription.cpp

```diff
@@ -633,6 +633,114 @@ TEST_F(TestSubscriptionUse, ignore_local_publications) {
   }
 }
 
+TEST_F(TestSubscriptionUse, ignore_local_publications_serialized) {
```

Comment:
this test makes sure to ignore the publication serialized message from local publisher. this test fails without having https://github.com/ros2/rmw_fastrtps/pull/823 for rmw_fastrtps.